docs/templating-language.md

```diff
@@ -1223,6 +1225,28 @@ Takes the given list of strings as a pipe and joins them on the provided string:
 {{ $items | join "," }}
 ```
 
+### `mergeMap`
+
+Takes the result from [`explode`](#explode) and an exploded argument then merges it both maps. The argument's source is will not be overridden by piped map.
+
+```golang
+{{ $base := tree "base" | explode }}
+{{ $overrides := tree "overrides" | explode | mergeMap $base}}
+{{ with $overrides }}
+{{ .a.b.c }}{{ end }}
+```
+
+### `mergeMapWithOverride`
+
+Takes the result from [`explode`](#explode) and an exploded argument then merges it both maps. The argument's source is will be overridden by piped map.
+
+```golang
+{{ $base := tree "base" | explode }}
+{{ $overrides := tree "overrides" | explode | mergeMap $base}}
```

Comment:
I think here you might have meant to use `mergeMapWithOverride` instead of `mergeMap`?

Reply:
Sure did thanks again for fixing

Reply:
Sure did. Fixed. Thanks again!